editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/restconfiguration/RestConfigEditor.java

```diff
@@ -357,14 +363,29 @@ private ToolBar createRestConfigurationToolbar(Composite parent) {
 		return toolbar;
 	}
 
+	private ToolBar createRestTabToolbar(Composite parent) {
+		ToolBarManager toolBarManager = new ToolBarManager(SWT.FLAT);
+		ToolBar toolbar = toolBarManager.createControl(parent);
+		toolbar.setBackground(Display.getCurrent().getSystemColor(SWT.COLOR_WHITE));
```

Comment:
you need to use the toolkit (FormToolkit) otherwise, it won't respect themes (black/white)

so either form.createToolbar if it exists or toolkit.adapt on the created element

Reply:
now that the adapt method is used, i think that this line setting the background needs to be removed.